File: app/src/main/java/com/ravisharma/playbackmusic/Model/Album.java
package com.ravisharma.playbackmusic.Model;

import android.net.Uri;

import com.google.gson.annotations.SerializedName;

public class Album {

    @SerializedName("albumId")
    private long albumId;

    @SerializedName("albumArt")
    private Uri albumArt;

    @SerializedName("albumName")
    private String albumName;

    @SerializedName("albumArtist")
    private String albumArtist;

    @SerializedName("numberOfSongs")
    private String numberOfSongs;

    public Album(long albumId, Uri albumArt, String albumName, String albumArtist, String numberOfSongs) {
        this.albumId = albumId;
        this.albumArt = albumArt;
        this.albumName = albumName;
        this.albumArtist = albumArtist;
        this.numberOfSongs = numberOfSongs;
    }

    public long getAlbumId() {
        return albumId;
    }

    public Uri getAlbumArt() {
        return albumArt;
    }

    public String getAlbumName() {
        return albumName;
    }

    public String getAlbumArtist() {
        return albumArtist;
    }

    public String getNumberOfSongs() {
        return numberOfSongs;
    }
}

File: data/src/main/java/com/ravisharma/playbackmusic/data/olddb/model/Album.kt
package com.ravisharma.playbackmusic.data.olddb.model

import android.net.Uri
import com.google.gson.annotations.SerializedName

data class Album(
        @field:SerializedName("albumId") val albumId: Long,
        @field:SerializedName("albumArt") val albumArt: Uri,
        @field:SerializedName("albumName") val albumName: String,
        @field:SerializedName("albumArtist") val albumArtist: String,
        @field:SerializedName("numberOfSongs") val numberOfSongs: String
)